packages/styles/settings-tools/_all-settings.scss

```diff
@@ -26,3 +27,4 @@
 @import './_s.flags.scss';
 @import './_s.stars.scss';
 @import './_s.tag.scss';
+@import './_u.mu-preview-grid.scss';
```

Comment:
Not in the all-settings, to avoid styles generations in all previews.
Import on demand

Reply:
fixed